zerver/views/auth.py

```diff
@@ -430,6 +447,11 @@ def api_fetch_api_key(request, username=REQ(), password=REQ()):
                                     realm_subdomain=get_subdomain(request),
                                     return_data=return_data)
     else:
+        if not ldap_auth_enabled():
```

Comment:
Ideally, in the case of subdomains, we'd pass a realm to `ldap_auth_enabled`, to make life easier when we eventually support per-realm configuration of LDAP auth.  Probably not a big deal if it's annoying to do, but I think it won't be.